docs/modules/writing-policies/pages/index.adoc

```diff
@@ -0,0 +1,158 @@
+
+=== e. Policy Metadata (WIP)
+
+In order to represent a policy as code that is compatible for EC/Conforma/OPA evaluation, we need to include a certain amount of metadata related to a policy. A standardized way of representing this required information is a policy metadata schema that each written policy should comply with. (For example, each policy should have a title and description)
+
+=== f. Policy Configuration
+
+The Conforma/EC tool used by Mequal relies on the usage of policy configurations that can point to a specific set of SCM repositories containing policies, so the tooling can fetch them for evaluation in the background. It also supports to include and exclude certain policies from being evaluated directly from the configuration file. It can also exist and be maintained as a Kubernetes resource.
+
+=== g. Evaluation
+
+An evaluation the output produced by EC/Conforma/OPA with Mequal after running a set of policies against a specific input. It can be in either json or yaml format and contains what violations or warnings have been raised against these policies for the input. This evaluation can be context-specific depending on the policies it runs and the type of input that is passed into the SBOM. (For example different policy checks for different SBOM versions, SBOM types, etc.)
+
+The violations help with our validation of the SBOMs to make sure they comply with the policy and fail the evaluation if there are any policies the SBOM doesn't comply with. These are more related to the basic important rules the SBOM must comply with to be considered valid.
+
+The warnings are more about giving feedback to related to the evaluation against certain policies without necessarily rejecting the SBOM input. They anticipate certain information that each SBOM may not necessarily have to represent, and provide guidance on how to include this information on the SBOM. It also tries to gauge the quality of the SBOM based on what it anticipates and that is actually found within the SBOM in the form of levels or points. 
+
+== 2. How to Write Policies
+
+=== Resources on Rego
+
+The policies evaluated against SBOMs using Mequal will need to be written using the https://www.openpolicyagent.org/docs/latest/policy-language/[Rego language], which is a language that is specialized in writing policies as code. So having a basic understanding of it will be necessary to write policies.
+
+=== Folder structures and corrsponding package hierarchy
+
+The first practice we should make sure to follow is to keep a directory of policies consistent with the rego package structure, very similar to how it's done in Java. So given we'd like to organize our policy like below:
+
+[source,bash]
+----
+sbom
+-> cyclonedx
+----> policy1.rego
+----> policy2.rego
+-> spdx
+----> policy1.rego
+----> policy2.rego
+----
+
+a .rego file representing sbom/cyclonedx/policy1.rego should have
```

Comment:
Put any files you reference in backticks.

Reply:
Done